gatewayconfig/bluetooth/characteristics/diagnostics_characteristic.py

```diff
@@ -33,10 +28,10 @@ def __init__(self, service, eth0_mac_address, wlan0_mac_address, firmware_versio
     def ReadValue(self, options):
         logger.debug('Read diagnostics')
         try:
-            p2pstatus = self.get_p2pstatus()
+            # p2pstatus = self.get_p2pstatus()
```

Comment:
removed this because it was using session bus.

Reply:
I guess this can be permanently removed for light because this relates to relaying I think?

Reply:
all traces of relaying will be removed as part of  https://github.com/NebraLtd/hm-diag/issues/322. Once https://github.com/NebraLtd/hm-pyhelper/pull/142 has been merged.